bench/commands/setup.py

```diff
@@ -148,6 +148,11 @@ def setup_requirements(node=False, python=False):
 		from bench.utils import update_node_packages
 		update_node_packages()
 
+@click.command("dev-requirements", help="Setup optional development dependencies")
```

Comment:
I think we don't need another command for this. This can be a part of the `setup-requirements` command.

Eg: `bench setup requirements --dev` to setup dev + prod requirements it for all bench apps; fulfilling Python and Node dependencies. `bench setup requirements --dev --python` for only the Python dependencies, etc.

Reply:
Not sure if we can handle `node`, as dev requirements are installed right now and changing that behaviour could be a breaking change. 😐 

Unlike python req.txt where we are proposing a new separate file node deps are all stored in `package.json`.

Reply:
We could probably though...not for Frappe, but for other apps. Else, we could just put a yellow line/comment saying `--dev` doesn't support fulfilling Node dev deps yet?

Reply:
Something along lines of "node dev deps are installed by default"